src/app/components/Form/Form.stories.tsx

```diff
@@ -9,8 +9,6 @@ export default {
 
 export const regular = (): JSX.Element => (
   <Form
-    nameLabel="Name des Schülers"
-    evaluationLabel="Worturteil"
     onSubmit={() => console.log('submitted')}
```

Comment:
- I guess, you are aware of these console.logs, right?

Reply:
Yes, it's because I still don't know how to handle the args prop of storybook: definitely something to look into in the future!